addons/website/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,139 @@
+---Models in module 'website'---
+obsolete model website.redirect (renamed to website.rewrite)
+new model website.rewrite (renamed from website.redirect)
+# DONE: pre-migration: renamed models
+
+new model website.route
+new model website.track
+new model website.visitor
+# NOTHING TO DO
+
+---Fields in module 'website'---
+website      / ir.ui.view               / track (boolean)               : NEW hasdefault
+website      / mrp.document             / key (char)                    : previously in module mrp
+website      / mrp.document             / website_id (many2one)         : previously in module mrp
+website      / mrp.document             / website_url (char)            : previously in module mrp
+website      / res.partner              / is_published (boolean)        : previously in module website_partner

```

Comment:
You should use openupgradelib's `update_module_moved_fields` method here (and in all moved fields). If not, there some garbage and side effect. See https://github.com/OCA/openupgradelib/commit/6a4360b353986cd2e606b7d1dcc2abfa74f683a6 where I have removed the non correct docstring.